### ISSUE TITLE
How do I make spells run actions upon landing?

### ISSUE THREAD

**Issue by mythguy1226** [elBukkit/MagicPlugin#612]
I am trying to create a spell that flings the player in the air and does an area of effect to damage others around them once they land. How would I assign actions to the event of the player landing?

**Comment by NathanWolf** [comment]
Unfortunately this is not currently possible. Adding some actions that run on land has been on the TODO list for quite a while.

**Comment by mythguy1226** [comment]
Awesome can't wait to see that feature in the future

**Comment by zaccieslak** [comment]
You could always add a delay that is timed for when the player lands. I don't know the velocity/power your spell has, but it shouldn't be too hard to measure the amount of time your spell has someone airborne in milliseconds! I made something similar to this, but it was horizontal velocity, not vertical. Just gotta get the timing down is all

**Comment by NathanWolf** [comment]
I just realized this is a duplicate of #303 so I'm going to close this one.
